test-lib-openshift.sh

```diff
@@ -14,7 +14,7 @@ OS_TESTSUITE_RESULT=1
 OS_CLUSTER_STARTED_BY_TEST=0
 
 function ct_os_cleanup() {
-  echo "${test_short_summary:-}"
+  echo "${TEST_SUMMARY:-}"
   if [ $OS_TESTSUITE_RESULT -eq 0 ] ; then
```

Comment:
```suggestion
  if [ TESTSUITE_RESULT -eq 0 ] ; then
```

Like here: https://github.com/sclorg/s2i-perl-container/blob/master/test/run#L15 or
https://github.com/sclorg/s2i-perl-container/blob/master/test/run#L405